### ISSUE TITLE
Importing categories?

### ISSUE THREAD

**Issue by agonalia** [jobisoft/CategoryManager#64]
Hi, there!
It could be possible to assign an specific field from an external adresses  book to a category when it is imported to Thunderbird?
Thank you!

**Comment by jobisoft** [comment]
Yes, the CatMan has a very powerful import/export feature. Use the context menu of a category (or the global "all" entry) in the category list in the lower left of the address book. You will be able to manually map the category field.

**Comment by agonalia** [comment]
Thanks, a lot, @jobisoft, but there may be something wrong with my TB release (78.7.1 64-bit)... or with me. ;) 
First, my adress book has not the same aspect as the one which screenshot you put in the addons page as a sample. I enclose an image of my own one. So, I cannot find how can I create or simply manage categories beyond a sightseeing of the list. This is why I cannot assign my labels to specific ones but to a default category ("Import_YYYY_MM_DDThh:mm:ss").
Second, I can import a .csv to that default group, indeed, but the result is a book with my >1000 adresses... completely empty! (???).
I think I have not the right TB version or something. Oh! Important: my O.S. is   Linux Mint 20.1.
Thanks, again.

Your sample screenshot:
![177293](https://user-images.githubusercontent.com/79252291/108343410-3328fc80-71dc-11eb-9196-90ecd3c77910.png)

Mine:
[Sin título 1.pdf](https://github.com/jobisoft/CategoryManager/files/6002077/Sin.titulo.1.pdf)
Import window:
[Sin título 2.pdf](https://github.com/jobisoft/CategoryManager/files/6002084/Sin.titulo.2.pdf)
